ui.tests/test-module/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "aem-cloud-wdio-sample-project",
```

Comment:
You should probably update these values. Also set `"private": true`, see https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private.

Reply:
I copied all this from venia but will fix it. ;-)